4806-document-different-ways-to-execute-queries-against-iox (#4850)
influxdata/docs-v2: content/influxdb/cloud-iox/query-data/tools/grafana.md

## Install Grafana

> Follow the [Grafana.com instructions to Install Grafana](https://grafana.com/docs/grafana/latest/setup-grafana/installation/) for your system.

Author: ```suggestion
Follow [Grafana installation instructions](https://grafana.com/docs/grafana/latest/setup-grafana/installation/)
for your operating system.
```

    - **WHERE**: To filter the query results, enter a conditional expression.
    - **GROUP BY**: To `GROUP BY` one or more fields or tags, enter them as a comma-delimited list.
                    If you include an aggregate function in the **SELECT** list,

>                     then you must include a time value in a `GROUP BY` or `PARTITION BY` clause.

Author: ```suggestion
                    then you must include one or more of the queried columns in a `GROUP BY` or `PARTITION BY` clause.
```
